MIGRATION_GUIDE.md

```diff
@@ -783,8 +783,8 @@ resource "auth0_role_permission" "write_posts_permission" {
 
 ### Client Grant Scopes
 
-The `scope` property on the `auth0_client_grant` resource has been renamed to `scopes` to reflect that multiple scopes
-can be managed per client grant.
+The `scope` property on the `auth0_client_grant` resource has been renamed to `scopes` to reflect the fact that multiple
```

Comment:
IMO "the fact" suggests that this can be debated but it is inherently factual.

```suggestion
The `scope` property on the `auth0_client_grant` resource has been renamed to `scopes` to reflect that multiple
```